modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts

```diff
@@ -100,26 +103,22 @@ export async function syncSite(page) {
 	});
 
 	await page
-		.getByTestId('Liferay DXP')
+		.getByTestId('Liferay')
```

Comment:
Hey @interaminense, this doesn't seem right to me. This ID will have the same value as the data source we are using, so if the AC has more than one data source, this will fail. In CI, the name of the data source is not the same as the local environment, it has the date that the test is being executed, so I don't think you can put the fixed value there.

You always need to select the first item in the list which will be the new property that is created after the connection. What do you think?

Reply:
Hey @ViniLopes20 it is already there, I just renamed because `Liferay DXP` was renamed to `Liferay` on master